src/qt/test/test_main.cpp

```diff
@@ -86,32 +86,32 @@ int main(int argc, char* argv[])
     app.setApplicationName("Bitcoin-Qt-test");
     app.createNode(*init);
 
+    int num_test_failures{0};
+
     AppTests app_tests(app);
-    if (QTest::qExec(&app_tests) != 0) {
-        fInvalid = true;
-    }
+    num_test_failures += QTest::qExec(&app_tests);
+
     OptionTests options_tests(app.node());
-    if (QTest::qExec(&options_tests) != 0) {
-        fInvalid = true;
-    }
+    num_test_failures += QTest::qExec(&options_tests);
+
     URITests test1;
-    if (QTest::qExec(&test1) != 0) {
-        fInvalid = true;
-    }
+    num_test_failures += QTest::qExec(&test1);
+
     RPCNestedTests test3(app.node());
-    if (QTest::qExec(&test3) != 0) {
-        fInvalid = true;
-    }
+    num_test_failures += QTest::qExec(&test3);
+
 #ifdef ENABLE_WALLET
     WalletTests test5(app.node());
-    if (QTest::qExec(&test5) != 0) {
-        fInvalid = true;
-    }
+    num_test_failures += QTest::qExec(&test5);
+
     AddressBookTests test6(app.node());
-    if (QTest::qExec(&test6) != 0) {
-        fInvalid = true;
-    }
+    num_test_failures += QTest::qExec(&test6);
 #endif
 
-    return fInvalid;
+    if (num_test_failures) {
+        qWarning("\nFailed tests: %d\n", num_test_failures);
```

Comment:
nit:
This is less of a suggestion and more of a preference. I don't think this line needs separate line breaks to make it stand out from other lines. I think we can forgo the "\n"s.

Reply:
Thanks @shaavan!  I like the line breaks to set off the summary, but if reviewers agree with you I'm ok to drop them.

Reply:
thanks @shaavan